README.md

```diff
@@ -1 +1,20 @@
 👤🎁🔄
+
+## About
+
+A static website that creates gift swop assignments and allows blacklisting of certain pairings from a list of user-provided names.
+The site generates a link to give to each person that states who they will be giving to.
+
+## Frequently Asked Questions
+### Why the British spelling `swop`?
+The term "swop" is a playful variation of "swap" commonly used in British English. We chose this spelling to add a touch of charm and uniqueness to the project, reflecting its lighthearted purpose of gift exchanges. Plus, it helps differentiate our platform while celebrating diverse language conventions!
```

Comment:
I like this take, but yeah...real answer is because original contributor/author is British. Its also a slight nod to avoid [Americentrism](https://en.wikipedia.org/wiki/Americentrism).

That also leads onto `why not secret santa`. This also to avoid is to a religious bias.

Reply:
HI! Thanks for the feedback. Yeah I honestly wasn't sure what the reason was, but I'll change it and add the other faq as well